sdk/ai/azure-ai-resources-autogen/dev_requirements.txt

```diff
@@ -0,0 +1,4 @@
+-e ../../../tools/azure-sdk-tools
```

Comment:
Is this a recommended best practice? A separate package folder for tsp configs?

Reply:
this was unintentionally included in in the PR. I've removed it